provider/core/src/data_provider.rs

```diff
@@ -24,7 +24,7 @@ use icu_locid::LanguageIdentifier;
 #[non_exhaustive]
 pub struct DataRequestMetadata;
 
-#[derive(Default, Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Default, Debug, PartialEq, Eq)]
```

Comment:
I think I see what's going on. You should not remove the `Clone` here.

Reply:
Ah right! Thanks for catching! Updated.